keras_hub/src/utils/keras_utils.py

```diff
@@ -104,3 +104,38 @@ def running_on_gpu():
         import torch
 
         return torch.cuda.is_available()
+
+
+def gpu_supports_fused_attention_op():
+    deny_list = ["t4"]
+    available_gpu = [gpu.upper() for gpu in get_gpu_name()]
+    for denied_gpu in deny_list:
+        if any(denied_gpu.upper() in gpu for gpu in available_gpu):
+            return False
+    return True
+
+
+def get_gpu_name():
```

Comment:
`get_gpu_names()`? And maybe move the `upper()` call into this function? You already do it for torch below but not the other backends which is a little confusing.